### PR TITLE
:bug: fix whitespace handling

### PR DESCRIPTION
**Pull request by ShuffleZZZ**

### CODE REVIEW

File: src/main/kotlin/com/github/pool_party/pull_party_bot/commands/PingCommandHandlers.kt [hunk 1]
```diff
@@ -38,7 +38,7 @@ fun Bot.handleStart(msg: Message) {
  * Return the help message.
  */
 fun Bot.handleHelp(msg: Message, args: String?) {
-    val parsedArgs = parseArgs(args)
+    val parsedArgs = parseArgs(args)?.distinct()
```
Comment (nothingelsematters) [inline src/main/kotlin/com/github/pool_party/pull_party_bot/commands/PingCommandHandlers.kt:41]: Shouldn't it be added in `parseArgs`?

File: src/main/kotlin/com/github/pool_party/pull_party_bot/commands/Utils.kt [hunk 2]
```diff
@@ -32,7 +32,8 @@ fun Bot.onAdministratorCommand(command: String, action: (Message, String?) -> Un
 fun Bot.modifyCommandAssertion(chatId: Long, name: String): Boolean =
     name.equals("admins").not().also { if (!it) sendMessage(chatId, ON_ADMINS_PARTY_CHANGE, "Markdown") }
 
-fun parseArgs(args: String?): List<String>? = args?.split(' ')?.map { it.trim().toLowerCase() }?.distinct()
+fun parseArgs(args: String?): List<String>? =
+    args?.split(' ')?.map { it.trim().toLowerCase() }?.filter { it.isNotBlank() }
```
Comment (nothingelsematters) [inline src/main/kotlin/com/github/pool_party/pull_party_bot/commands/Utils.kt:36]: Probably it is better to write `args?.let { <other code> }` in order not to write `?.` constantly. But that's ok

**Review by nothingelsematters — APPROVED** [review]

File: src/main/kotlin/com/github/pool_party/pull_party_bot/commands/PingCommandHandlers.kt (hunk 1 above)
Comment (ShuffleZZZ) [reply src/main/kotlin/com/github/pool_party/pull_party_bot/commands/PingCommandHandlers.kt:41]: party pulling needs all arguments with repetition to handle `/create user-name user-name` case

Comment (nothingelsematters) [reply src/main/kotlin/com/github/pool_party/pull_party_bot/commands/PingCommandHandlers.kt:41]: 🤪

**Pull request merged**